src/hooks/useGamificationBadges.ts

```
@@ -0,0 +1,63 @@
+    queryKey: ["gamification_badges", publishedOnly],
+    queryFn: async () => {
+      let query = supabase.from("gamification_badges" as any).select("*").order("created_at", { ascending: false });
+      if (publishedOnly) query = query.eq("is_published", true);
+      const { data, error } = await query;
+      if (error) throw error;
+      return (data || []) as unknown as GamificationBadge[];
+    },
+  });
+
+  const createBadge = useMutation({
+    mutationFn: async (badge: { title: string; description: string; svg_payload_json: BadgeComposition }) => {
```

ESLint

[error] Replace `·title:·string;·description:·string;·svg_payload_json:·BadgeComposition` with `⏎······title:·string;⏎······description:·string;⏎······svg_payload_json:·BadgeComposition;⏎···`

(prettier/prettier)